bin/posydon-run-pipeline

```diff
@@ -619,6 +623,13 @@ def logic_rerun(grid, rerun_type):
             if np.isnan(iniY) or iniY<0.96:
                 runs_to_rerun += [i]
         runs_to_rerun = np.array(runs_to_rerun)
+    elif rerun_type == 'timestep_MTlimit':
+        termination_flags = TF1_POOL_ERROR
+    elif rerun_type == 'egrav_Pgas_dXdt_terms':
+        termination_flags = TF1_POOL_ERROR #['reach cluster timelimit']
+        new_mesa_flag = {'include_composition_in_eps_grav': '.true.',
+                         'change_initial_lnPgas_flag': '.true.',
+                         'new_lnPgas_flag': '.true.'}
```

Comment:
Usually, we should a change of more than one inlist parameter into a MESA-inlist PR. The reason is, that all the changes put here will end up in the directory names, which become too long for the file system in case of too many parameters.

Reply:
OK thanks. I've now moved these inlist options to inlist1/inlist2 in a separate branch linked above and updated the branch referenced for this rerun in the pipeline script

I've also added an `x_logical_ctrl(7)` switch to control whether timestep_MTlim is enabled or not.